eogrow/utils/meta.py

```diff
@@ -73,8 +72,10 @@ def get_os_import_path(import_path: str) -> str:
     """
     module_spec = importlib.util.find_spec(import_path)
     if module_spec is not None and module_spec.origin is not None:
+        if module_spec.origin == "frozen":  # python 3.11 optimizations preload certain core modules
+            raise ValueError(f"Given import path {import_path!r} belongs to a 'frozen' module, no path available.")
         return module_spec.origin
-    raise ValueError(f"Given import path {import_path} not found")
+    raise ValueError(f"Given import path {import_path!r} not found")
```

Comment:
So, this does't really change anything, since we don't use import paths from builtins anyway, it just failed because of one particular test which used `os`, but the failed test itself wasn't really important, it just coincidentally used the `os` package which was affected?

Reply:
yes, but I believe it is something that should be handled 'just in case', otherwise we'd need to add it to docs somewhere etc. Since this happens at validation time, i dont mind it failing if it detects issues.

I was also thinking that maybe we should remove this functionality... It was added as a way to make evalscripts available on the cluster, but i think we can come up with something better